src/stream-notes/Dockerfile

```diff
@@ -4,17 +4,31 @@ ARG BUILDVERSION=0.0.0
 
 WORKDIR /app
 
-COPY ./src/stream-notes .
-COPY ./src/common ./src
+# Copy stream-notes dependency files
+COPY ./src/stream-notes/package.json ./src/stream-notes/package-lock.json ./
 
-WORKDIR /app/src
+# Clean install stream-notes depdenencies
+RUN npm ci --silent
 
-RUN npm install
+# Copy common dependency files
+COPY ./src/common/package.json ./src/common/package-lock.json ./
```

Comment:
Did we forget to change the WORKDIR to /app/src?

When you run 'npm ci --silent' it removes all the packages from stream-notes' package.json an installs common's